api/src/main/java/org/apache/iceberg/transforms/VoidTransform.java

```diff
@@ -49,6 +49,14 @@ public Void apply(S t) {
 
   private VoidTransform() {}
 
+  /**
+   * Transforms a value to its corresponding partition value.
+   *
+   * @param value a source value
+   * @return a transformed partition value
```

Comment:
nit: We could mention that this method always return null.

Reply:
```suggestion
   * @return null
```